src/main/java/pl/piekoszek/gorskimatches/challange/JudgeResult.java

```diff
@@ -0,0 +1,39 @@
+package pl.piekoszek.gorskimatches.challange;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class JudgeResult {
+
+    public Result getResultForChallengeUser1(int scoreUser1, int scoreUser2, float timeUser1, float timeUser2) {
```

Comment:
How do I know which user is winner or loser?
Mayby instead od `WINNER` `LOSER` use `PLAYER_1_WIN`, `PLAYER_2_WIN`, or sth similar.